### PR TITLE
fix: add more context to profile log messages

### PR DESCRIPTION
**Pull request by maxbrunet**
* Add `PID` (and `addr` if available) to profile log messages, this will help to investigate processes the agent is having trouble with
* Add an empty target to systemd target groups, this will look less odd in some places like the `parca_agent_sd_discovered_targets{config="all"}` metric reporting `0` targets
* ~~Fix ignoring perf map not found errors (#750)~~

### CODE REVIEW

File: pkg/symbol/symbol.go [hunk 1]
```diff
@@ -90,10 +90,7 @@ func (s *Symbolizer) resolveJITedFunctions(pid profiler.PID, locations []*profil
 		if !ok {
 			sym, err := perfMap.Lookup(loc.Address)
 			if err != nil {
-				if !errors.Is(err, perf.ErrNotFound) {
```
Comment (maxbrunet) [inline pkg/symbol/symbol.go:93]: `Lookup()` only only returns an `ErrNoSymbolFound` error

Comment (kakkoyun) [reply pkg/symbol/symbol.go:93]: I think the `ErrNoSymbolFound` sentinel error only gets checked here. Let's remove the error if we decide not to check against it.

**Review by kakkoyun — CHANGES_REQUESTED** [review]
Good catch!

**Comment by kakkoyun** [comment]
Let's reconsider this since #756 also fixes the same problem.

**Comment by maxbrunet** [comment]
> Let's reconsider this since #756 also fixes the same problem.

@kakkoyun I've removed the overlapping changes

File: pkg/symbol/symbol.go (hunk 1 above)
Comment (maxbrunet) [reply pkg/symbol/symbol.go]: For the record, I think it is a best practice to declare all errors, so leaving `ErrNoSymbolFound` even if nothing catches it at the moment is good

I usually enable the [go-err113](https://github.com/Djarvur/go-err113) linter which disallow dynamic errors.

**Review by kakkoyun — APPROVED** [review]

File: pkg/symbol/symbol.go (hunk 1 above)
Comment (kakkoyun) [reply pkg/symbol/symbol.go]: We can declare it, but we don't need to export it. If the identifier is exported, it gives me the sense that it's used outside the package.

**Pull request merged**